src/libs/ajax/leonardo/models/runtime-config-models.ts

```diff
@@ -0,0 +1,70 @@
+export interface GpuConfig {
+  gpuType: string
+  numOfGpus: number
+}
+
+export type CloudService = 'GCE' | 'DATAPROC' | 'AZURE_VM'
```

Comment:
Wondering if we want to adjust the name of this.
Maybe CloudComputeService?
I could see someone thinking CloudService is AZURE | GCP

Reply:
Hmm.. it's true. A bit misleading since the type name is sourced directly from the backend field value name. Backend does not name it well however... I'll call it `ComputeType` in the UI. Its not really related to the cloud at all, anymore than a MachineType/Region is.

Reply:
That sounds like a great name for it!